02/2_array/main.c

```diff
@@ -0,0 +1,17 @@
+#include <stdio.h>
+
+int main(void)
+{
+    int grades[5] = {70, 60, 85, 55, 90};

```

Comment:
put a space between opening braces and the first element. Same with the closing braces